tcp-app-starters-common/src/main/java/org/springframework/cloud/stream/app/tcp/ContentTypeEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.tcp;
+
+import java.util.Properties;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.core.env.PropertiesPropertySource;
+
+/**
+ * An {@EnvironmentPostProcessor} to set the {@code spring.cloud.stream.bindings.output.contentType}
+ * property to {@code application/octet-stream} if it has not been set already.
+ *
+ * @author Chris Schaefer
+ */
+public class ContentTypeEnvironmentPostProcessor implements EnvironmentPostProcessor {
```

Comment:
This class must be replaced by the `ContentTypeEnvironmentPostProcessor` from the `core`.
See https://github.com/spring-cloud-stream-app-starters/scriptable-transform/commit/243da380d33229f11fdf18f9caf17c2b5f38a5b1

Reply:
+1